generators/pattern-library/configure.js

```diff
@@ -111,19 +110,69 @@ function init( options ) {
 
     // **** SETTINGS ****
 
-    if ( settings.dest ) {
+    if ( settings.dest && settings.dest === 'none' ) {
+
+        settings.dest = null;
+
+    }
+    else if ( settings.dest ) {
+
         settings.dest = path.resolve( settings.cwd, settings.dest );
+
+    }
+    else {
+
+        error = error || [];
```

Comment:
Make error an array from the very beginning, and push errors as configure runs. Then we check for the length of the error array to determine if there are errors in our test.